lib/puppet-lint/lexer.rb

```diff
@@ -362,7 +362,7 @@ def interpolate_string(string, line, column)
             lexer.tokens.each do |token|
               tok_col = column + token.column + (ss.pos - contents.size - 1)
               tok_line = token.line + line - 1
-              tokens << new_token(token.type, token.value, token.value.size, :line => tok_line, :column => tok_col)
+              tokens << new_token(token.type, token.value, token.value.size + 3, :line => tok_line, :column => tok_col)
```

Comment:
What does this do on a line w/o the curlies? e.g., ` "A double quoted $string without properly bracketed variables"`